manifest_staging/charts/aad-pod-identity/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Deploy components for aad-pod-identity
 name: aad-pod-identity
-version: 4.0.0
+version: 4.1.0
```

Comment:
nit: this change isn't required as we update the version before the release but not a blocker for this PR